documentation/user_kit.md

```diff
@@ -23,6 +23,13 @@ Partners can fetch a list of all the doors that are enabled for them, by using a
 	GET https://rest.latchaccess.com/access/sdk/v1/doors
 	```
 
+	HTTP Query Parameters
+
+	```
+	pageSize: <integer>   (by default returns all doors)
+	pageToken: "<string>" (default is "1", first page)
```

Comment:
@usmanaz I believe we need to add next_page_token to the response body as well and let's mention the expected value when there is no next page.

Reply:
Good catch. I also added the expected value to the fetch users response. Thanks!

Reply:
thanks!